### DISCUSSION TITLE
Palm Rejection

### DISCUSSION THREAD

**Discussion by keiwando** [keiwando/toonsquid#13] [General]
While palm rejection is something that the iPad should perform automatically, especially when using Apple Pencil, I would like to use this thread to keep track of palm rejection failures that inevitably come up here and there.

Please report under what circumstances you usually notice the palm rejection failing and what result it has (e.g. unwanted brush strokes, unwanted undo/redo, unwanted canvas movement etc.). 
Thank you!

**Comment by trinhpxl** [comment]
unwanted canvas movement is when brush strokes happen or my project flips thru the frames !

**Comment by TruddyTheDuddi** [comment]
Unwanted undo/redo s happen the most frequently for me
